src/west/configuration.py

```diff
@@ -186,11 +195,11 @@ def __init__(self, topdir: PathType | None = None):
     def get_paths(self, location: ConfigFile = ConfigFile.ALL):
         ret = []
         if self._global and location in [ConfigFile.GLOBAL, ConfigFile.ALL]:
-            ret.append(self._global.path)
+            ret += self._global.paths
         if self._system and location in [ConfigFile.SYSTEM, ConfigFile.ALL]:
-            ret.append(self._system.path)
+            ret += self._system.paths
         if self._local and location in [ConfigFile.LOCAL, ConfigFile.ALL]:
-            ret.append(self._local.path)
+            ret += self._local.paths
```

Comment:
Is this related to this new feature? If not, separate commit please to reduce mental load. Reviewers don't want to guess.

Reply:
For clarity it might be better to use `extend` here, not blocking though.